packages/core/scripts/sass-inline-svg.js

```diff
@@ -0,0 +1,12 @@
+const inliner = require('sass-inline-svg');
+
+module.exports = {
+    // Sass function to inline a UI Icon svg and change its path color:
+    // svg("16px/icon-name.svg", (path: (fill: $color)) )
+    'svg': inliner('../../resources/icons', {
+        // run through SVGO first
```

Comment:
if we run this through SVGO, does it use all the default settings? Are those okay? sometimes SVGO gets a little overzealous.

Reply:
yeah default settings and i don't think there's a way to configure. seems sufficient for now.

if we care about the settings then we should add a script to ensure the tracked icon files are already optimized.